### PR TITLE
feat(collaboration): display separated for open and closed issues and pagination added

### LINKED ISSUE #851: Separate display of open and closed issues
## Motivation

I'm usually interested in focusing on open issue, and seeing closed issues there together is a distracting.

## Proposal

* Separate open and closed issues (see mockup below).
* Add pagination to support a large number of issues

## Context

image

## Extras

Think about adding a higher-order component  (or some other framework support) for easy application of pagination.

### PR DESCRIPTION
**Pull request by vfried**
The issues are now split in Open and Complete and on top of that i implemented the pagination. We could with little effort add issues search also, i'm not sure if this is something we want now.

image

Regarding the style of the new nav bar i added... I left a comment in the merge request improvements issue...
https://github.com/SwissDataScienceCenter/renku-ui/issues/854#issuecomment-630189696

TODO: add a message when there aren't any issues inside the project.. 
Closes #851

### CODE REVIEW

**Comment by vfried** [comment]
Shall we change Completed to Closed????

--> An issue can be closed without being completed...  Same happens with merge requests.

File: src/issue/list/IssueList.state.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+      initialized: false,
+      errorMessage: "",
+      loading: false
+    };
+    this.setObject(newData);
+    return newData;
+  }
+
+  setQueryAndSortInSearch( pathName, pageNumber, issuesState) {
+    this.setPathName(pathName);
```
Comment (ciyer) [inline src/issue/list/IssueList.state.js:89]: How are the issues sorted? I think they should be sorted by most recently modified.

**Comment by vfried** [comment]
I just made a "CollaborationList" component that makes the state and container generic, so it can be used in merge requests and issues. In case we need this for some other component we could think about making it more generic but i'm not sure it's the best option.

Now both merge requests and issues have open/closed and pagination.

File: src/issue/list/IssueList.state.js (hunk 1 above)
Comment (vfried) [reply src/issue/list/IssueList.state.js]: I forgot to rename that function, I am not setting sorting there, we can though sort by last modified.

Comment (vfried) [reply src/issue/list/IssueList.state.js]: I just did this, changed the name of this function but also sorted the elements by last update.

**Pull request merged**